pjsip/src/pjsip/sip_msg.c

```diff
@@ -979,10 +978,10 @@ static pjsip_generic_array_hdr* pjsip_generic_array_hdr_clone( pj_pool_t *pool,
                                                  const pjsip_generic_array_hdr *rhs)
 {
     unsigned i;
-    pjsip_generic_array_hdr *hdr = PJ_POOL_ALLOC_T(pool, pjsip_generic_array_hdr);
+    pjsip_generic_array_hdr *hdr = pjsip_generic_array_hdr_create(pool, &rhs->name);
```

Comment:
Similar to above.

Reply:
`pjsip_generic_array_hdr_create()` does the same by calling `pjsip_generic_array_hdr_init()` which allocate new space in the provided `pj_pool_t` for the header name of the cloned header via `pj_strdup()` (pjsip/src/pjsip/sip_msg.c:939).  
The problem doesn't exist with header which are not *PJSIP_H_OTHER* because the header name points to a static memory space defined by `const pjsip_hdr_name_info_t pjsip_hdr_names[];`.